pulsar-common/src/main/java/org/apache/pulsar/common/policies/data/ClusterData.java

```diff
@@ -20,18 +20,48 @@
 
 import static com.google.common.base.Preconditions.checkNotNull;
 
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
 import java.util.LinkedHashSet;
 import java.util.Objects;
 
 import com.google.common.base.MoreObjects;
 
+@ApiModel(
+    value = "ClusterData",
+    description = "The configuration data for a cluster"
+)
 public class ClusterData {
+    @ApiModelProperty(
+        name = "serviceUrl",
+        value = "The http rest service url (for admin operations)",
```

Comment:
HTTP, URL
When we use the abbreviation form for protocol names or proper nouns in descriptive text, we use the uppercase letters.